person.rb

```diff
@@ -19,3 +26,39 @@ def of_age?
     false
   end
 end
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable
+  end
```

Comment:
- [ ] The `correct_name` should return the result of the correct_name method of the @namable, kindly make sure to fix this so it returns the correct data

 ```suggestion
  def correct_name
    @nameable.correct_name
  end
```